File: titov/src/Controller/IndexController.php
<?php

namespace App\Controller;

use App\Entity\Visit;
use App\Repository\VisitRepository;
use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Request;
use Symfony\Component\HttpFoundation\Response;

class IndexController extends AbstractController
{
    public function index(Request $request, VisitRepository $repository): Response
    {
        $visit = new Visit();
        $visit->setIp($request->getClientIp());
        $repository->persist($visit);

        return new Response('Symfony application. <a href="/api/visits">All visits</a>');
    }
}

File: ride/src/main/java/ovh/testdomain/ride/controllers/IndexController.java
package ovh.testdomain.ride.controllers;

import io.swagger.v3.oas.annotations.Hidden;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;
import ovh.testdomain.ride.Main;
import ovh.testdomain.ride.models.Visit;
import ovh.testdomain.ride.repositories.VisitRepository;

import javax.servlet.http.HttpServletRequest;

@RestController
public class IndexController {
    private final VisitRepository repository;

    public IndexController(VisitRepository repository) {
        this.repository = repository;
    }

    @Hidden
    @GetMapping("/")
    public String index(HttpServletRequest request) {
        Visit visit = new Visit();
        visit.setIp(request.getRemoteAddr());
        repository.save(visit);

        StringBuilder sb = new StringBuilder();
        sb
                .append("Spring Boot application v.")
                .append(Main.class.getPackage().getImplementationVersion())
                .append("<br>")
                .append("<a href='/actuator'>App status</a>")
                .append("<br>")
                .append("<a href='/swagger-ui.html'>Swagger</a>");
        return sb.toString();
    }
}
